config.json

```diff
@@ -1065,6 +1067,8 @@
     {
       "uuid": "1818f134-0ed9-e680-9b29-45ffd2b3344b0f841c7",
       "slug": "markdown",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Can you make a PR for these changes? I know these keys only matter for Nextercism, but I'd rather have them fixed sooner rather than later.

Reply:
Done. #1110